contrib/core-contract-tests/tests/pox-4/pox_StackAggregationCommitAuthCommand.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  logCommand,
+  PoxCommand,
+  Real,
+  Stub,
+  Wallet,
+} from "./pox_CommandModel.ts";
+import { poxAddressToTuple } from "@stacks/stacking";
+import { expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+
+/**
+ * The `StackAggregationCommitAuthCommand` allows an operator commits partially
```

Comment:
NIT, change:

'allows an operator commits partially stacked STX..."
to
'allows an operator to partially commit stacked STX & to allocate...'